sdk-base/src/main/java/com/mapbox/maps/extension/observable/model/DataSourceType.kt

```diff
@@ -1,38 +1,31 @@
 package com.mapbox.maps.extension.observable.model
 
-import com.google.gson.annotations.SerializedName
-
 /**
  * Describes data source of request for resource-request event.
  */
-enum class DataSourceType {
+enum class DataSourceType(val value: String) {
   /**
    * data source as resource-loader.
    */
-  @SerializedName("resource-loader")
```

Comment:
We need to retain `@SerializedName` it would change the output for users that convert model objects to json